CMakeLists.txt

```diff
@@ -138,6 +138,14 @@ if(${SUBSURFACE_TARGET_EXECUTABLE} MATCHES "DesktopExecutable")
 	else()
 		set(SUBSURFACE_TARGET subsurface)
 	endif()
+	if (USE_WEBENGINE)
+        add_definitions(-DUSE_WEBENGINE)
+        list(APPEND QT_EXTRA_COMPONENTS WebEngine WebEngineWidgets)
+        list(APPEND QT_EXTRA_LIBRARIES Qt5::WebEngine Qt5::WebEngineWidgets)
+    else()
+        list(APPEND QT_EXTRA_COMPONENTS WebKitWidgets)
+        list(APPEND QT_EXTRA_LIBRARIES Qt5::WebKitWidgets)
+    endif()
```

Comment:
This means that we no longer are able to build with neither QtWebEngineWidget - that's why Travis fails